### DISCUSSION TITLE
Up to date tags? (and PyPI)

### DISCUSSION THREAD

**Discussion by marianolazzaro** [gbeced/basana#12] [Q&A] [answered]
Hi, I've seen that the latest commits (with bug fixes) haven't got a tag or a release, and that PyPI version is the latest tag, which is superseded by later commits.

Is the latest commit the best to use this software right now?

Thanks!

**Comment by gbeced** [comment, marked as answer]
Hi, I just released version 1.3.2.
Thanks for letting me know.
